schema.sql

```diff
@@ -1,5 +1,21 @@
 /* Database schema to keep the structure of entire database. */
 
+/*
+POSTGRESQL
+Create a table named animals with the following columns:
+    id: integer
+    name: string
+    date_of_birth: date
+    escape_attempts: integer
+    neutered: boolean
+    weight_kg: decimal
+*/
+
```

Comment:
- [ ] Please kindly remove commented code from your code base as it makes your code base dirty and less readable.

Reply:
Thank You @Mihndim2020